code/datums/storage/storage.dm

```diff
@@ -287,35 +288,51 @@
 /// ~Lemon
 GLOBAL_LIST_EMPTY(cached_storage_typecaches)
 
-/datum/storage/proc/set_holdable(list/can_hold_list, list/cant_hold_list)
-	if(!isnull(can_hold_list) && !islist(can_hold_list))
-		can_hold_list = list(can_hold_list)
-	if(!isnull(cant_hold_list) && !islist(cant_hold_list))
-		cant_hold_list = list(cant_hold_list)
-
+/**
+ * Sets what type of contents this storage supports
+ * Arguments
+ *
+ * * list/can_hold_list - The list of item types whitelisted in this storage rejecting everything else
+ * * list/cant_hold_list - The list of item types blacklisted in this storage accepting everything else
+ * * list/exception_hold_list - The list of items that can exceed `max_specific_storage`. It can only fit `exception_count` of such items
+ */
+/datum/storage/proc/set_holdable(list/can_hold_list, list/cant_hold_list, list/exception_hold_list)
+	can_hold = null
```

Comment:
Just want to make sure that this won't cause us problems down the line - have you checked and ensured that nothing relies on this? i.e. that there aren't any subtypes that rely on their parent's can/can't hold lists, because this would change that behavior and could cause unexpected issues.

Reply:
No problems as of yet. All the instances I have seen where both parent and child call this proc is in overriding the list values I.e parent would set can_hold list to some value but child type would override that value with something else.

There aren't any instances like for e.g. parent would set can_hold list but child type would set cant_hold list so it's all good